### PR TITLE
UIUtils downloadImage unit tests

### PR DESCRIPTION
**Pull request by spoorthipujariadobe**
## Description

Unit tests for UIUtils.downloadImage()

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/gradle.properties [hunk 1]
```diff
@@ -16,7 +16,7 @@ org.gradle.caching=true
 android.useAndroidX=true
 
 moduleName=messaging
-moduleVersion=3.2.1
+moduleVersion=3.3.0
```
Comment (prudrabhat) [inline code/gradle.properties:19]: nit: can we do this in a separate PR. Since we are dealing with snapshots it would be clearer have a good demarcation. Also, I think you can run the update_versions actions on this branch also...

**Review by prudrabhat — APPROVED** [review]

File: code/gradle.properties (hunk 1 above)
Comment (spoorthipujariadobe) [reply code/gradle.properties]: Done

**Review by prudrabhat — APPROVED** [review]

**Review by prudrabhat — APPROVED** [review]
